.github/workflows/ci.yml

```diff
@@ -218,17 +316,35 @@ jobs:
             ${{ steps.cache-key.outputs.buildtool-monthly-branch-cache-key }}-
             ${{ steps.cache-key.outputs.buildtool-monthly-cache-key }}-
 
+      - id: create_database
+        uses: loiclefevre/test@v1.0.18
```

Comment:
Please switch to using the hash of the version and document the version as comment like we do with all the other steps as well.